### PR TITLE
[PTDT-1115] User can create a relationship feature

### PR DESCRIPTION
**Pull request by lgluszek**
Add`Tool.Type.RELATIONSHIP`, which when passed to OntologyBuilder, will create a relationship tool.

How to test:

```python
from labelbox import Client, Tool, OntologyBuilder, MediaType

client = Client(api_key="...", endpoint="http://localhost:8080/graphql")

object_features = [
    Tool(
        tool=Tool.Type.RELATIONSHIP,
        name="relationship",
        color="#ff0000",
    )
]

ontology_builder = OntologyBuilder(
    tools=object_features
)

ontology = client.create_ontology("Ontology from new features", ontology_builder.asdict(), media_type=MediaType.Image)
```

When run, will create a new ontology with a relationship tool

image

### CODE REVIEW

**Comment by karenkyang** [comment]
looks good to me

File: labelbox/schema/ontology.py [hunk 1]
```diff
@@ -241,6 +241,7 @@ class Type(Enum):
         BBOX = "rectangle"
         LINE = "line"
         NER = "named-entity"
+        RELATIONSHIP = "edge"
```
Comment (kkim-labelbox) [inline labelbox/schema/ontology.py:244]: Could you update the notebooks with `Tool.Type.RELATIONSHIP`? For instance, I see `ontologies.ipynb` lists out all possible Tool types

Comment (lgluszek) [reply labelbox/schema/ontology.py:244]: updated 4 notebooks:
- examples/annotation_import/image.ipynb
- examples/annotation_import/pdf.ipynb
- examples/basics/ontologies.ipynb
- examples/prediction_upload/video_predictions.ipynb

File: examples/annotation_import/pdf.ipynb [hunk 2]
```diff
@@ -173,7 +173,10 @@
         "  tools=[ \n",
```
Comment (kkim-labelbox) [inline examples/annotation_import/pdf.ipynb:173]: <p>Tool Type should RELATIONSHIP here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/921/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

**Comment by kkim-labelbox** [comment]
One small comment. Also, looks like there are some merge conflicts

File: examples/annotation_import/pdf.ipynb (hunk 2 above)
Comment (lgluszek) [reply examples/annotation_import/pdf.ipynb:124]: thanks https://github.com/Labelbox/labelbox-python/pull/921/commits/69373ed1423c9d32c7b2de9332fa4fcd91260d64

**Pull request merged**